docs/source/open-software-week/index.md

```diff
@@ -0,0 +1,107 @@
+# Open Software Week
+
+We are excited to announce our inaugural **NIU Open Software Week**, taking
+place in **August 2025** in **London, UK**. This event will bring together researchers, developers, and users of open-source software for some hands-on training, community-building and hacking.
+
+:::{admonition} Save the Date!
+:class: important
+
+**August 11-15 2025**, [Sainsbury Wellcome Centre](https://maps.app.goo.gl/CzWFFjXJZwX87aMj6)
+
+- **April 7th 2025**: Registration opens
+- **May 11th 2025**: Travel grant application deadline
+- **June 31st 2025**: Registration closes
```

Comment:
@niksirbi 30th June or 31st July? 😂

Reply:
Sorry meant to be last day of June, so June 30th.